apps/server/src/modules/authorization/rules/course.rule.spec.ts

```diff
@@ -1,12 +1,14 @@
 import { Test, TestingModule } from '@nestjs/testing';
+import { Course, User } from '@shared/domain/entity';
+import { Permission } from '@shared/domain/interface';
```

Comment:
Going through all of the changes, I found lots of instances of, e.g.

```
import type { User } from '../../../shared/domain/entity';
import { Course } from '../../../shared/domain/entity';
```

whereas @SevenWaysDP has asked multiple times to replace it with

```
import type { User } from '@shared/domain/entity';
import { Course } from '@shared/domain/entity';
```

Would it make sense to replace / change all such imports as @SevenWaysDP requested, because I still did find some more instances, which were not marked for a change (e.g. `apps/server/src/modules/authorization/rules/course.rule.ts`)?